CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed 
 
 - `pangea.audit_dump` only dump before events if the leaf_index is not None.
+- exposed `vault_parameters` and `llm_request` fields to redact api
```

Comment:
Should be moved up into a new `### Fixed ` under `## Unreleased`. v5.2.0 has already been released so this change cannot be listed under v5.2.0.